maxim/logger/portkey/__init__.py

```
@@ -0,0 +1,15 @@
+enabling automatic logging of OpenAI-compatible calls via Maxim.
+"""
+
+from .client import instrument_portkey
+from .portkey import (
+    MaximPortkeyClient,
+)
+
+__all__ = [
+    "instrument_portkey",
+    "MaximPortkeyClient",
+]
```

Ruff (0.11.9)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)